docs/en/reference/working-with-objects.rst

```diff
@@ -250,6 +250,10 @@ as follows:
 -  If X is a detached entity, an exception will be thrown on
    flush.
 
+.. caution::
+
+    Do not pass detached entities to the persist operation.
```

Comment:
Thank you for the addition to the docs. You actually could use the same sentence for the caution-block.

```suggestion
    Do not pass detached entities to the persist operation. The persist operation always
    considers entities that are not yet known to this EntityManager as NEW.
```

Reply:
Right, it's more clear and spares from having to go read the comment in-code 👍